src/main/resources/application.properties

```diff
@@ -1 +1,5 @@
-server.port={8001}
+spring.datasource.url=jdbc:postgresql://localhost:5432/coursesdb
```

Comment:
Create variable for
database name
database port
database url

Reference all these variables inside of spring.datasource.url